addons/mail/static/src/models/messaging.js

```diff
@@ -259,7 +259,7 @@ Patch({
             for (const guest of this.models["Guest"].all()) {
                 guestIds.push(guest.id);
             }
-            this.env.services["im_status"].registerToImStatus("res.partner", partnerIds);
+            //            this.env.services["im_status"].registerToImStatus("res.partner", partnerIds);
```

Comment:
Ok for comment, just keep single whitespace between `//` and beginning of the statement.